README.md

```diff
@@ -20,24 +20,91 @@ Specification for binding services to runtime applications running in Kubernetes
 
 Main section of the doc.  Has sub-section that outline the design.
 
-### 1.  Making a service bindable
+### Making a service bindable
 
-*  What metadata is needed for a service to be bindable (e.g. name of fields it supports, name of secret it produces, etc)
-*  Where is this information located
+#### Recommended
+For a service to be bindable it **should** provide:
+* a ConfigMap that contains the name of the Secret holding the binding data (see the [Service Binding Schema](#service-binding-schema) section below), and describes metadata associated with each of the items referenced in the Secret, as well as a reference to this ConfigMap.  If there are different Secrets created for different CRs, then this ConfigMap should specify the pattern of the generated Secrets.
 
-### 2.  Service Binding Schema
+This pattern ensures the binding data is properly secured in a Secret and has corresponding metadata to enhance the consumption experience of the binding. 
 
-*  What is the schema of a Service Binding object / structure.
-*  What metadata goes along that describes it
+The key/value pairs insides this ConfigMap are:
+* A single `Secret=<name_of_secret>` - where `<name_of_secret>` is the qualified name (including the namespace) of the k8s Secret (or the pattern of Secrets), which contains the binding data for this service.
+* A set of `Metadata.<property>` - where `<property>` maps to one of the defined keys for this service, and `<value>` represents the description of the value.  For example, this is useful to define what format the `password` key is in, such as apiKey, basic auth password, token, etc.
 
-### 3.  Request service binding
 
-*  How do we request a binding from a service (assume the service has been provisioned)
+#### Minimum
+If not following the recommended path above, for a service to be bindable it **must** provide:
+* a Secret that contains the binding data and a reference to this Secret, OR, map its `status` properties to the corresponding binding data.
```

Comment:
This isn't very clear. Some elaboration / examples would be welcome.

Reply:
I updated that section. Please let me know if it's more clear now.